fabtests/prov/efa/Makefile.include

```diff
@@ -51,3 +54,15 @@ prov_efa_src_fi_efa_rnr_queue_resend_LDADD = libfabtests.la
 prov_efa_src_fi_efa_info_test_SOURCES = \
 	prov/efa/src/efa_info_test.c
 prov_efa_src_fi_efa_info_test_LDADD = libfabtests.la
+
+efa_exhaust_mr_reg_srcs = \
+	prov/efa/src/efa_exhaust_mr_reg_common.h \
+	prov/efa/src/efa_exhaust_mr_reg_common.c
+
+if HAVE_VERBS_DEVEL
+prov_efa_src_fi_efa_exhaust_mr_reg_rdm_pingpong_SOURCES = \
+	prov/efa/src/efa_exhaust_mr_reg_rdm_pingpong.c \
+	$(efa_exhaust_mr_reg_srcs) \
+	${benchmarks_srcs}
```

Comment:
```suggestion
	$(benchmarks_srcs)
```

(Also I'm assuming `benchmark_srcs` is defined in the parent Makefile)

Reply:
Yes